Orbitersdk/samples/ProjectApollo/src_sys/toggleswitch.cpp

```diff
@@ -1140,6 +1142,9 @@ bool CircuitBrakerSwitch::CheckMouseClickVC(int event, VECTOR3 &p)
 
 double CircuitBrakerSwitch::Voltage()
 {
+	GetState();
+	if (Failed) return 0.0;			//If a braker has failed, the braker will not supply voltage
+	
 	if ((state != 0) && SRC)
```

Comment:
The two lines above are not necessary. Just change "state" to "GetState()" and it does the same thing.